### PR TITLE
Update go xorm tests

### PR DESCRIPTION
**Pull request by fimac**
- updates proxy, and goeql
- adds additional tests for jsonb ops that we have in main currently and inline with poc
- deletes examples because they are a duplication of what we have in the e2e tests.
- updates to download eql from releases and install on init of docker
- removes dropping and recreating db when running tests  not necessary.
Screenshot 2024-10-30 at 12 42 08 PM

### CODE REVIEW

File: languages/go/xorm/run.sh [hunk 1]
```diff
@@ -26,28 +26,9 @@ subproject_teardown() {
   docker compose down
 }
 
-subproject_examples() {
```
Comment (fimac) [inline languages/go/xorm/run.sh:29]: Removed this command as no longer necessary (see comment in description). The tests are the source of truth for examples. No need to duplicate.

File: languages/go/xorm/migrations.go [hunk 2]
```diff
@@ -3,29 +3,8 @@ package main
...
 
-func InstallEql(engine *sql.DB) {
-	log.Println("Start installing custom types and function")
-	installDsl(engine)
-
-}
-
-// Installing EQL
-func installDsl(engine *sql.DB) {
-	path := "../../../release/cipherstash-encrypt-dsl.sql"
```
Comment (fimac) [inline languages/go/xorm/migrations.go:17]: removed as this is now being installed on init of docker with a download directly from eql

File: languages/go/xorm/main.go [hunk 3]
```diff
@@ -213,57 +175,15 @@ func main() {
...
-	setupDb()
-
 	// Connect to go test directly and create table
 	createTable()
 
-	// Install EQL and add config
-	installEql()
-}
-
-func runExamples() {
```
Comment (fimac) [inline languages/go/xorm/main.go:234]: As noted in the description, these examples are duplicates of the e2e tests. Removing as there's no need for both, tests are the source of truth for working examples.

File: languages/go/xorm/main.go [hunk 4]
```diff
@@ -123,44 +123,6 @@ func (eb *EncryptedBoolField) FromDB(data []byte) error {
 	return nil
 }
 
-func setupDb() {
```
Comment (fimac) [inline languages/go/xorm/main.go:126]: Unnecessary to have this. The db is created when docker starts

**Review by auxesis — APPROVED** [review]
Big fan of these simplifications. 

Nice work @fimac ❤️

**Pull request merged**